nvflare/private/fed/app/utils.py

```diff
@@ -56,7 +56,7 @@ def kill_child_processes(parent_pid, sig=signal.SIGTERM):
         return
     children = parent.children(recursive=True)
     for process in children:
-        process.send_signal(sig)
```

Comment:
Thanks for the fix, should we remove this "sig" argument since it is not used?

Reply:
removed.